l5kit/l5kit/data/zarr_dataset.py

```diff
@@ -27,6 +29,7 @@
 FRAME_DTYPE = [
     ("timestamp", np.int64),
     ("agent_index_interval", np.int64, (2,)),
+    ("tl_faces_index_interval", np.int64, (2,)),
```

Comment:
`traffic_light_faces_index_interval`: I do not want use abbreviation in zarr's fieldname

Reply:
Ok, I'll do it even if I don't agree with it xD but I won't be able to check if everything works after that

Reply:
@lucabergamini thanks ;) I will create a new l5kit wheel and update avresearch side now